libzhlgen/test/functions/win32/1.6.9/freetype.zhl

```diff
@@ -15,6 +15,17 @@ static cleanup uint64_t freetype::easy_printCenter_DO_NOT_USE_DIRECTLY(int fontS
 ".8d4c240483e4f0ff71fc5589e55756535181ec????????8b5110":
 static cleanup __cdecl void freetype::easy_printAutoShrink(int fontId, float x, float y, int maxWidth, bool centered, const std::string& text);
 
+"55660fefc9660fefc089e55389cb83e4f083ec20":
+__thiscall void freetype::font_data::init(freetype::font_data *this, const void *buffer, int bufferSize, unsigned int h, bool glow);
+"5589e583e4f083ec108b4104890424e8????????c9c3":
+__thiscall void freetype::font_data::clean(freetype::font_data *this);
+"5589e5565383e4f083ec308b4d08f30f104d0c85c97e??3b0d????????7d??803d????????008b15????????74??8b1d????????39d97c??8d349d000000008b143285d274??89c829d88db60000000039c389c17f??8b143229d885d275??660fefc0eb??8b1c8a660fefc085db74??0f2ec173??0f2ec876??0f2e4b247a??75??f30f10432c":
+cleanup __cdecl float font_baseline(int font_id, float size);
+"5589e5565383e4f083ec308b4d08f30f104d0c85c97e??3b0d????????7d??803d????????008b15????????74??8b1d????????39d97c??8d349d000000008b143285d274??89c829d88db60000000039c389c17f??8b143229d885d275??660fefc0eb??8b1c8a660fefc085db74??0f2ec173??0f2ec876??0f2e4b247a??75??f30f104328":
+cleanup __cdecl float font_height(int font_id, float size);
+
+"!89108b559cc705(???????\?)000000408950048b55a0":
+float *Global_freetype_sil_freetype_outline;
```

Comment:
In Linux this comes after `font_baseline` (so already fine if other comments were followed)
Linux 64:
```
"!.f30f5905(???????\?)4531c031c9f30f2df0":
offsetVariable float* Global_freetype_sil_freetype_outline;
```

Linux 32: (direct address match, use like Win32)
`!.f30f5905(???????\?)c744240801000000f30f2dc0`